seeders/20200614165815-file_constants.js

```diff
@@ -0,0 +1,211 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface) => {
+    const rolesCount = await queryInterface.sequelize.query(
+      "SELECT count(*) from file_constants;"
+    );
+    if (rolesCount[0][0].count > 0) {
+      return rolesCount;
+    } else {
+      return queryInterface.bulkInsert("file_constants", [
+        {
```

Comment:
It should be better if we can declare these file constants in separate file and can import it wherever needed.